src/content/blog/astro-290.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: "Astro 2.9: View Transitions (experimental)"
+description: "Astro 2.9 is out with experimental support for View Transitions, allowing smooth page transitions."
+publishDate: "July 20, 2023"
+authors:
+  - matthew
+coverImage: ""
+socialImage: ""
+lang: "en"
+---
+
+Astro 2.9 is out and features experiment support for [view transitions](https://developer.chrome.com/docs/web-platform/view-transitions/), as well as redirects configuration and improved script bundling.
+
+- **[View Transitions (experimental)](#view-transitions-experimental)**
+- **[Redirects](#redirects)**
+- **[Hoisted script optimization](#hoisted-script-optimization)**
+
+If you already have Astro installed, you can upgrade it to 2.9 by running the `upgrade` command in your project (using your package manager of choice):
```

Comment:
```suggestion
If you already have Astro installed, you can upgrade it to 2.9 by running the upgrade command in your project (using your package manager of choice):
```
removing the inline code because you use `install` for npm, so let's just use the English phrase "upgrade command" vs making it code.